tests/phpunit/elementor/modules/landing-pages/test-module.php

```diff
@@ -0,0 +1,37 @@
+<?php
+namespace Elementor\Testing\Modules\LandingPages;
+
+use Elementor\Testing\Elementor_Test_Base;
+use Elementor\Modules\LandingPages\Module;
+
+class Elementor_Test_Landing_Pages_Module extends Elementor_Test_Base {
+	/** @var Module */
+	private static $module;
+
+	public function test_should_confirm_module_activation() {
+		// Initialize an instance of the module being tested.
+		self::$module = new Module();
+
+		$this->assertDocumentTypeRegistered( Module::DOCUMENT_TYPE );
+	}
+
+	public function test_should_return_landing_pages() {
+		$this->assertEquals( 'landing-pages', self::$module->get_name() );
+	}
+
+	// TODO: move the following method to the documents trait.
+	/**
+	 * Asserts that a type is registered in documents
+	 *
+	 * @param string $type_name
+	 */
+	private function assertDocumentTypeRegistered( $type_name ) {
```

Comment:
Move this method to a more general location

Reply:
Done. Created a trait called Elementor_Library and put it in `tests/phpunit/traits`